src/common/currency-util.ts

```diff
@@ -6,3 +6,20 @@ export const getCurrencySymbol = (currency: string) => {
   }
   return CurrencyIcons[currency]?.symbol || "";
 };
+
+export const getShortCurrencyNumber = (value: string) => {
+  const n = Number(value);
+  if (Math.abs(n) < 1e3) {
+    return `${n}`;
+  }
+  if (Math.abs(n) >= 1e3 && Math.abs(n) < 1e6) {
+    return `${(n / 1e3).toFixed(0)}K`;
```

Comment:
we should probably just use https://www.npmjs.com/package/short-number. otherwise, 1230 will be shortened to 1k